lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -4626,6 +4835,63 @@ impl FunctionCodeGenerator<CodegenError> for X64FunctionCode {
                     self.machine.release_temp_gpr(tmp_out);
                 }
             }
+            Operator::I32TruncSSatF32 => {
+                let loc =
+                    get_location_released(a, &mut self.machine, self.value_stack.pop().unwrap());
+                let ret = self.machine.acquire_locations(
+                    a,
+                    &[(WpType::I32, MachineValue::WasmStack(self.value_stack.len()))],
+                    false,
+                )[0];
+                self.value_stack.push(ret);
+
+                let tmp_out = self.machine.acquire_temp_gpr().unwrap();
+                let tmp_in = self.machine.acquire_temp_xmm().unwrap();
+
+                Self::emit_relaxed_binop(
+                    a,
+                    &mut self.machine,
+                    Assembler::emit_mov,
+                    Size::S32,
+                    loc,
+                    Location::XMM(tmp_in),
+                );
+                Self::emit_f32_int_conv_check_sat(
+                    a,
+                    &mut self.machine,
+                    tmp_in,
+                    -2147483904.0,
+                    2147483648.0,
```

Comment:
Names and/or explanations for numbers like these are always appreciated

Reply:
Added a section of constants to the bottom of the file. (Same in the LLVM backend too.) Done.